stdlib/Test/src/Test.jl

```diff
@@ -963,8 +965,22 @@ mutable struct DefaultTestSet <: AbstractTestSet
     showtiming::Bool
     time_start::Float64
     time_end::Union{Float64,Nothing}
+    failfast::Bool
 end
-DefaultTestSet(desc::AbstractString; verbose::Bool = false, showtiming::Bool = true) = DefaultTestSet(String(desc)::String, [], 0, false, verbose, showtiming, time(), nothing)
+function DefaultTestSet(desc::AbstractString; verbose::Bool = false, showtiming::Bool = true, failfast::Union{Nothing,Bool} = nothing)
+    if isnothing(failfast)
+        # pass failfast state into child testsets
+        parent_ts = get_testset()
+        if parent_ts isa DefaultTestSet
+            failfast = something(parent_ts.failfast, false)
```

Comment:
```suggestion
            failfast = parent_ts.failfast
```
`something` here seems to be redundant?

Reply:
True. The `fallback_testset` isn't a `DefaultTestSet` so the first testset is set to false by the other branch. Thanks